UnicornRender/video/source/CameraProjection.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#include <unicorn/video/CameraProjection.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+
+CameraProjection::CameraProjection(system::Window* window, glm::mat4& cameraProj) :
+    m_aspect(static_cast<float>(window->GetSize().first) / window->GetSize().second),
+    m_cameraProjection(cameraProj),
+    m_pWindow(window)
```

Comment:
put all commas before each member so that when you add more variables, diff will be shorter